tools/documentation_crawler/documentation_crawler/spiders/common/spiders.py

```diff
@@ -143,6 +143,7 @@ def _make_requests(self, url: str) -> Iterator[Request]:
             url in ["http://localhost:9981/", "http://localhost:9981"]
             or url.startswith("http://localhost:9981/#")
             or url.startswith("http://localhost:9981#")
+            or url.startswith("http://localhost:9981/communities")
```

Comment:
I removed `/communities` from the check that was failing. @timabbott if you have concerns that this could cause regressions in the future since this is the only landing page for which we are making this exception, I can comment out the code in HTML which results in this error. (I don't want to delete it though)

Reply:
Seems fine but we should add a comment.